doc/cabal-package.rst

```diff
@@ -1020,139 +1020,13 @@ For more information about the rationale and some examples, see
 ..
     TODO inline the blog post
 
-
-Opening an interpreter session
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-While developing a package, it is often useful to make its code
-available inside an interpreter session. This can be done with the
-``repl`` command:
-
-.. code-block:: console
```

Comment:
Are you sure that all this block contains no useful information? I am checking [current manual for `repl`](https://cabal.readthedocs.io/en/stable/cabal-commands.html#cabal-repl)

Reply:
I do agree that this is useful information, but I think it is not in the correct place to write it down. It looks to me that section 6.2 wants to be the reference section for all the things that can be written in the package description/.cabal file. For some sections (such as executable, testsuite and benchmark) of the cabal file it makes sense to refer to the documentation of `cabal run` `cabal test` and `cabal bench`, because these commands are directly related to the subsections.  But commands such as `cabal repl` and `cabal get` which were also documented in section 6.2 do not correspond to a specific section of the cabal file, but to general workflows with `cabal-install` instead. A command such as `cabal repl` should be documented in the command reference section, and maybe also in the sections 1+2+3 which describe general cabal workflows.